### DISCUSSION TITLE
Welcome to spikingjelly Discussions!

### DISCUSSION THREAD

**Discussion by fangwei123456** [fangwei123456/SpikingFlow#35] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZulunZhu** [comment]
Hello, I am interested that how could we cite your work if we develop projects based on your work? Do you have any publication papers related to spikingjelly?

**Reply by fangwei123456** [reply]
Thanks! We will add the citation of this repo into readme.md in hours.

**Reply by fangwei123456** [reply]
We have added the citation bib into readme.md. Thanks for your citation!

**Reply by fangwei123456** [reply]
There was a bug in the previous citation bib in readme.md. We have fixed it.
